src/FbTk/TextBox.cc

```diff
@@ -591,7 +591,7 @@ void TextBox::adjustPos(){
 }
 
 
-void TextBox::select(std::string::size_type pos, int length)
+void TextBox::select(std::string::size_type pos, unsigned length)
```

Comment:
How does this trigger a compiler warning? And also the next lines make clear that length is supposed to be signed.